Sources/DataPipeline/DataPipelineConfigOptions.swift

```diff
@@ -39,6 +39,9 @@ public struct DataPipelineConfigOptions {
         self.writeKey = writeKey
     }
 
+    /// Site Id that will be used for tracking data migration
+    public let siteId: String = ""
```

Comment:
how is this being set?

Reply:
Will be set by the customer, have added a default value though. Is there something else that you expect?

Reply:
I think Shahroz is correct here and customers might not be able to set it because the constructor above only takes `writeKey`?

Reply:
Ah! Yes, right! 🤦

Reply:
My understanding is that we expect the customer to set the value of siteId using config like

```
CustomerIO.initialize(writeKey: writeKey, logLevel: logLevel) { config in
            config.siteId = "site_id_value_from_customer"
 }
```

Yes, the other way is to get the value as a parameter but I am not sure if we want that ? Let me know if we are not on same page ! I have no preferences and can make the changes if required.

Reply:
but it is `let` ?

Reply:
Ah ! Yes, you are right. Fixed in next commit